lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :basket, :menu
+
+  def initialize
+    @menu = {
+      :Margherita => 4.99,
+      :Pepperoni => 6.99,
+      :"Four Cheese" => 5.99,
+      :Sausage => 6.99,
+    }
+    @basket = {} # item name and quantity
+    @sum_total = 0
+    @time = (Time.now + 3600).strftime "%H:%M"
+  end
+
+  def order(food, amount = 1)
+    raise "We do not sell that item" unless @menu.has_key?(food.to_sym)
+    @basket[food.to_sym] = amount
+  end
+
+  def total
```

Comment:
Try to avoid using single letter vars unless they're some well known mathematical constants, perhaps.  Here, you could change `k, v` for `item, quantity` or something like that.